core/src/main/java/com/amplifyframework/api/GraphQlBehavior.java

```diff
@@ -139,6 +140,29 @@
             @NonNull Consumer<ApiException> onFailure
     );
 
+    /**
+     * Perform a GraphQL query against a configured GraphQL endpoint.
+     * This operation is asynchronous and may be canceled by calling
+     * cancel on the returned operation. The response will be provided
+     * to the `onResponse` callback.  If there is data present
+     * in the response, it will be cast as the requested class type.
+     * Requires that only one API is configured in your
+     * `amplifyconfiguration.json`. Otherwise, emits an ApiException to
+     * the provided `onFailure` callback.
```

Comment:
These new docs should start off with some strongly indicative language "PAges! How does a page work? You can use a page ... "